test/dynamo/test_error_messages.py

```diff
@@ -446,13 +446,7 @@ def fn(x):
         first_graph_break = next(iter(counters["graph_break"].keys()))
         self.assertExpectedInline(
             first_graph_break,
-            """\
-Attempted to call function marked as skipped
-  Explanation: Dynamo cannot trace optree C/C++ function optree._C.PyCapsule.flatten.
-  Hint: Consider using torch.utils._pytree - https://github.com/pytorch/pytorch/blob/main/torch/utils/_pytree.py
-
-  Developer debug context: module: optree._C, qualname: PyCapsule.flatten, skip reason: <missing reason>
-""",
+            "Attempted to call function marked as skipped",
```

Comment:
Can you change this test (and the other one) to look at the graph break logs instead of counters? We still want to be checking for the full error message here.

Reply:
Thanks for your review! But the `counter` instance does not store the full error message. I cannot get it here. Do you have any ideas? Should we add an another dict to store it? Thanks!

Reply:
You can look directly into logging records using `@make_logging_test(graph_breaks=True)`

Reply:
Thanks! Made a fix in this commit: https://github.com/pytorch/pytorch/pull/153818/commits/e4e7ef8739f826a6fa1eff486bea1e1e83fa34a2, please take another look when you have time.